src/acquisition/covidcast/covidcast_row.py

```diff
@@ -0,0 +1,270 @@
+from dataclasses import asdict, dataclass, field, fields
+from datetime import date
+from typing import Any, ClassVar, Dict, Iterable, List, Optional
+
+import pandas as pd
+from delphi_utils import Nans
+
+from delphi.epidata.acquisition.covidcast.csv_importer import CsvImporter
+from delphi.epidata.server.utils.dates import day_to_time_value, time_value_to_day
+from delphi.epidata.server.endpoints.covidcast_utils.model import PANDAS_DTYPES
```

Comment:
🔥 
At first I was like "is that allowed???" but then I looked at the acquisition environment on the server and hot damn it is!

Reply:
Yea, it seems like all the delphi-epidata code can get server imports, but the server can't import non-server code (mostly because of unnecessarily specific Docker build instructions).